### PR TITLE
holdings: issue creation permission management

### PR DESCRIPTION
**Pull request by zannkukai**
* Closes https://github.com/rero/rero-ils/issues/1705.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html [hunk 1]
```diff
@@ -125,7 +125,7 @@ <h1 class="mb-2" translate>Holdings</h1>
   <!-- END OF DETAILS TAB ---------------------------------------------------->
 
   <!-- LOCAL FIELDS TAB ------------------------------------------------------------>
-  <tab *ngIf="showhideLocalFieldsTab" id="holding-local-field-tab" tabOrder="3">
+  <tab *ngIf="displayLocalFieldsTab" id="holding-local-field-tab" tabOrder="3">
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html:128]: [permissions]="[PERMISSIONS.LOFI_SEARCH, PERMISSIONS.LOFI_CREATE]"

Remove `ngIf` and use `permissions` directive

File: projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.ts [hunk 2]
```diff
@@ -56,16 +60,13 @@ export class SerialHoldingDetailViewComponent implements OnInit {
...
 
-  /**
-   * Show or hide local fields tab
-   * @return boolean - if False, hide the local fields tab
-   */
-  get showhideLocalFieldsTab(): boolean {
+  /** Determine if the `Local fields` tab should be displayed or not. */
+  get displayLocalFieldsTab(): boolean {
     return this._permissionsService.canAccess([PERMISSIONS.LOFI_SEARCH, PERMISSIONS.LOFI_CREATE]);
   }
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.ts:71]: Remove this

File: projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.ts [hunk 3]
```diff
@@ -185,5 +160,49 @@ export class SerialHoldingDetailViewComponent implements OnInit {
...
+   *   holding. This last check could be determined checking the holding
+   *   `update` permissions.
+   */
+  private _computePermissions(): void {
+    if (this._permissionsService.canAccess(PERMISSIONS.ITEM_CREATE)) {
+      this._recordPermissionService
+        .getPermission('holdings', this.holding.metadata.pid)
+        .subscribe((data: RecordPermissions) => {
+          this.recordPermissions = data;
+          this.allowIssueCreation = data?.update?.can;
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.ts:178]: Does `data?.update?.can` return false, if there is no value?

Comment (zannkukai) [reply projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.ts:178]: you're right ! I will change my code.

**Review by jma — APPROVED** [review]
Can you add more details in your commit message?

**Review by Garfield-fr — APPROVED** [review]

**Comment by Garfield-fr** [comment]
Typo in your commit message `Closes @rero/rero-ils#1705.` to `Closes rero/rero-ils#1705.`

**Comment by PascalRepond** [comment]
## PO tests

- [ ] REP: Using a pro_all_permissions, the buttons are still displayed.
- [ ] REP: holdings detailed view: The can_delete message when hovering over an item is now empty. Should say that the resource doesn't belong to my library.

**Review by PascalRepond — APPROVED** [review]

**Pull request merged**